### PR TITLE
[NUI] Open new GridLayout APIs and Properties

### PR DESCRIPTION
**Pull request by neostom432**
### Description of Change ###
Open new GridLayout APIs and Properties

### API Changes ###
https://code.sec.samsung.net/jira/browse/TCSACR-352

### CODE REVIEW

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs [hunk 1]
```diff
@@ -236,17 +235,17 @@ public static void SetHorizontalAlignment(View view, Alignment value)
...
         public static void SetVerticalAlignment(View view, Alignment value)
         {
             SetChildValue(view, VerticalAlignmentProperty, value);
         }
 
         /// <summary>
-        /// [Draft] The Distance between Column
+        /// The Distance between Column
         /// </summary>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
```
Comment (dasg34) [inline src/Tizen.NUI/src/public/Layouting/GridLayout.cs:247]: I think we can remove this kind of comment.

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs [hunk 2]
```diff
@@ -304,7 +300,7 @@ public GridLayout()
         }
 
         /// <summary>
-        /// [Draft] Get/Set the number of columns in the GridLayout should have.
+        /// Get/Set the number of columns in the GridLayout should have.
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/Layouting/GridLayout.cs:303]: Grammar. There is inconsistency with the description of the Rows property "Get/Set the number of rows in the grid" which has the same logic. Also please use 3rd person to describe a property "Gets or sets the number..."

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs [hunk 3]
```diff
@@ -261,10 +259,9 @@ public float ColumnSpacing
         }
 
         /// <summary>
-        /// [Draft] The Distance between Rows
+        /// The Distance between Rows
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/Layouting/GridLayout.cs:262]:
```suggestion
        /// The distance between rows.
```

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs [hunk 4]
```diff
@@ -236,17 +235,16 @@ public static void SetHorizontalAlignment(View view, Alignment value)
...
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public static void SetVerticalAlignment(View view, Alignment value)
         {
             SetChildValue(view, VerticalAlignmentProperty, value);
         }
 
         /// <summary>
-        /// [Draft] The Distance between Column
+        /// The Distance between Column
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/Layouting/GridLayout.cs:245]:
```suggestion
        /// The distance between columns.
```

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs [hunk 5]
```diff
@@ -22,7 +22,7 @@
 namespace Tizen.NUI
 {
     /// <summary>
-    /// [Draft] This class implements a grid layout
+    /// This class implements a grid layout
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/Layouting/GridLayout.cs:25]: Please write something more, because its purpose is already suggested in the class name.

Comment (neostom432) [reply src/Tizen.NUI/src/public/Layouting/GridLayout.cs]: DONE

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs (hunk 3 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/Layouting/GridLayout.cs]: DONE

File: src/Tizen.NUI/src/public/Layouting/GridLayout.cs (hunk 2 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/Layouting/GridLayout.cs]: DONE

**Pull request merged**